README.md

```diff
@@ -182,3 +182,8 @@ In your fork of this repository (not a consuming repository):
       `update` driver and the convention subdirectories themselves. Of
       note, `${CONVENTION_ROOT}/_lib/` contains some utilities that may
       be useful for `update`s.
+
+To test your changes, you can use `BOILERPLATE_GIT_REPO` environment 
+variable and set it to you local clone in order to overide the version of 
+boilerplate used. As `update` is cloning the git repo, ensure to have your 
+changes locally committed for your testing. 
```

Comment:
Couple of typos:

```suggestion
To test your changes, you can use the `BOILERPLATE_GIT_REPO` environment 
variable and set it to your local clone in order to overide the version of 
boilerplate used. As `update` is cloning the git repo, ensure you have your 
changes locally committed for your testing.
```

There's another variable, `BOILERPLATE_GIT_CLONE`, that you would use if you wanted to e.g. add flags -- or even, for local testing, use a totally different command (like `rsync` :P). Do you think it's worth documenting that?

Also, an example would be great. Especially one that clones from my local machine, as I don't know how to do that :P